lib/YaST/EFItools.pm

```diff
@@ -0,0 +1,41 @@
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+package YaST::EFItools;
```

Comment:
(minor thing) I would suggest to use always lower letter, in my experience it complicates things (as we don't have proper auto-complete code in editor por Perl).

Reply:
VSCode does have proper autocomplete for that.

Reply:
I think that it suggests you names that you have already typed or in exists in the file, but it is not like when you press the dot " . " and you get the properties of the objects with documentation (like in Playwright with typescript). In Perl unless you have your setup configure in a particular way that I don't know anybody that really figure out properly, perhaps in tools team, I'm not able to to get that. But nevermind, in the code is a minor thing, for test names is a bit messy I remember debugging that sometimes I got wrong the capitalization accidentally, so better to put all in lower letter, ie.: install_SLES. But feel free to close this topic, it is fine as it is.